skywalking/plugins/sw_redis.py

```diff
@@ -27,6 +27,101 @@
 }
 note = """"""
 
+OPERATIONS_WRITE = set({'GETSET',
+                        'SET',
+                        'SETBIT',
+                        'SETEX ',
+                        'SETNX ',
+                        'SETRANGE',
+                        'STRLEN ',
+                        'MSET',
+                        'MSETNX ',
+                        'PSETEX',
+                        'INCR ',
+                        'INCRBY ',
+                        'INCRBYFLOAT',
+                        'DECR ',
+                        'DECRBY ',
+                        'APPEND ',
+                        'HMSET',
+                        'HSET',
+                        'HSETNX ',
+                        'HINCRBY',
+                        'HINCRBYFLOAT',
+                        'HDEL',
+                        'RPOPLPUSH',
+                        'RPUSH',
+                        'RPUSHX',
+                        'LPUSH',
+                        'LPUSHX',
+                        'LREM',
+                        'LTRIM',
+                        'LSET',
+                        'BRPOPLPUSH',
+                        'LINSERT',
+                        'SADD',
+                        'SDIFF',
+                        'SDIFFSTORE',
+                        'SINTERSTORE',
+                        'SISMEMBER',
+                        'SREM',
+                        'SUNION',
+                        'SUNIONSTORE',
+                        'SINTER',
+                        'ZADD',
+                        'ZINCRBY',
+                        'ZINTERSTORE',
+                        'ZRANGE',
+                        'ZRANGEBYLEX',
+                        'ZRANGEBYSCORE',
+                        'ZRANK',
+                        'ZREM',
+                        'ZREMRANGEBYLEX',
+                        'ZREMRANGEBYRANK',
+                        'ZREMRANGEBYSCORE',
+                        'ZREVRANGE',
+                        'ZREVRANGEBYSCORE',
+                        'ZREVRANK',
+                        'ZUNIONSTORE',
+                        'XADD',
+                        'XDEL',
+                        'DEL',
+                        'xtrim'})
```

Comment:
`xtrim`'s name style is different from others. Please confirm it.

Reply:
Thanks, I have confirmed it.